applications/nrf_desktop/configuration/nrf52840_pca10059/ble_scan.h

```diff
@@ -13,5 +13,21 @@
  */
```

Comment:
please add "_def" postfix to the file name

Reply:
done

Reply:
done